File: src/MyLibrary/MVVM/Attivita/UltimaAttivitaVm.cs
using MyLibrary.MVVM.Cliente;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace MyLibrary.MVVM.Attivita
{
    public class UltimaAttivitaVm
    {
        public int AttivitaNo { get; set; }
        public string Descrizione { get; set; }
        public double Ore { get; set; }
        public string Data { get; set; }
        public string DescCliente { get; set; }
        public string DescCommessa { get; set; }
        public string DescFattura { get; set; }
        public string DescLocazione { get; set; }
        public string DescPagamento { get; set; }
        public string DescTipo { get; set; }
        public string DescProgetto { get; set; }

        public ClienteVm Cliente { get; set; }

    }
}

File: src/Aurelia/App/attivita/ultimaAttivitaVm.ts
import {Cliente} from '../clienti/Cliente';

export class UltimaAttivitaVm {
    public AttivitaNo: number;
    public Descrizione: string;
    public Ore: number;
    public Data: string;
    public DescCliente: string;
    public DescCommessa: string;
    public DescFattura: string;
    public DescLocazione: string;
    public DescPagamento: string;
    public DescTipo: string;
    public DescProgetto: string;

    public Cliente: Cliente;
}